network/p2p/scoring/registry_test.go

```diff
@@ -387,8 +421,13 @@ func TestPersistingInvalidSubscriptionPenalty(t *testing.T) {
 			}
 		}))
 
+	ctx, cancel := context.WithCancel(context.Background())
+	signalerCtx := irrecoverable.NewMockSignalerContext(t, ctx)
+	reg.Start(signalerCtx)
```

Comment:
For all these tests, we should first ensure that the registry has started and is in `Ready` mode.

Reply:
https://github.com/onflow/flow-go/pull/5084/commits/af54b59ab391fdd326d003ec23b237c5e6d97f2c